File: PHP PowerPoint Examples/SlidesApiDemo/deleteSlidesList.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\SlidesApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new SlidesApi($configuration);

$name = "DeleteSlides_1.pptx"; 
$password = null; 
$folder = "/Slides/DeleteSlides/"; 
$storage = null; 
$slides = "9,5,3,59";
$apiInstance->deleteSlidesList($name, $slides, $password, $folder, $storage);

File: Python PowerPoint Examples/SlidesApiDemo/deleteSlidesList.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.slides_api.SlidesApi(configuration)

fileName = "DeleteSlides.pptx"
password = ""
storage = ""
slides = "9,5,3,59"
inputFolder = "input"
result =api.delete_slides_list(fileName, slides=slides, password=password, storage=storage, folder=inputFolder)